web/viewer.html

```diff
@@ -509,11 +509,11 @@
             <div id="markAsDecorative">
               <div class="radio">
                 <div class="radioButton">
-                  <input type="radio" id="decorativeButton" name="altTextOption">
+                  <input type="radio" id="decorativeButton" name="altTextOption" aria-describedby="decorativeDescription">
```

Comment:
With the other suggestion taken into account.
```suggestion
                  <input type="radio" id="decorativeButton" name="altTextOption" aria-describedby="decorativeLabel">
```